src/Datadog.Trace/RuntimeMetrics/RuntimeEventListener.cs

```diff
@@ -23,16 +27,38 @@ internal class RuntimeEventListener : EventListener, IRuntimeMetricsListener
         private static readonly string[] CompactingGcTags = { "compacting_gc:true" };
         private static readonly string[] NotCompactingGcTags = { "compacting_gc:false" };
 
+        private static readonly IReadOnlyDictionary<string, string> MetricsMapping;
+
         private readonly IDogStatsd _statsd;
 
         private readonly Timing _contentionTime = new Timing();
+
+        private readonly string _delayInSeconds;
+
         private long _contentionCount;
 
         private DateTime? _gcStart;
 
-        public RuntimeEventListener(IDogStatsd statsd)
+        static RuntimeEventListener()
+        {
+            MetricsMapping = new Dictionary<string, string>
+            {
+                ["current-requests"] = MetricsNames.CurrentRequests,
+                ["failed-requests"] = MetricsNames.FailedRequests,
+                ["total-requests"] = MetricsNames.TotalRequests,
+                ["request-queue-length"] = MetricsNames.RequestQueueLength,
+                ["current-connections"] = MetricsNames.CurrentConnections,
+                ["connection-queue-length"] = MetricsNames.ConnectionQueueLength,
+                ["total-connections"] = MetricsNames.TotalConnections
+            };
+        }
+
+        public RuntimeEventListener(IDogStatsd statsd, int delay)
```

Comment:
Can you specify the units for `delay`? Maybe in the parameter name (like `_delayInSeconds`) or using xml comments to document the ctor? Or even switch the type to `TimeSpan`?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1074/commits/5ed0ba4effebc9ea07000a7024aab83f1e8e1f88